src/main/java/com/robinpowered/sdk/model/Identifier.java

```diff
@@ -72,6 +72,9 @@ public static Interface fromString(String value) {
     private final String value;
     private final DateTime createdAt;
 
+    // Mutable
+    private Integer deviceId;
```

Comment:
Actually I just realized that `deviceId` is not on the remote identifier model at all:

```
        {
          "urn": "urn:robin-uuid:238EAAF3-BCED-494F-9375-074E2069FD27",
          "interface": "robin-uuid",
          "value": "238eaaf3bced494f9375074e2069fd27",
          "created_at": "2016-06-28T16:07:37+0000"
        }
```

It's only present for the iBeacon identifier interface. 

So really we probably shouldn't have it on this model at all, and you should maybe for now just infer it from the `:id` you used in the request path.